apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerDataReader.cs

```diff
@@ -177,7 +177,7 @@ public T GetFieldValue<T>(string columnName)
         /// </summary>
         /// <param name="i">The index of the column whose value will be returned.</param>
         /// <returns>The raw protobuf as a <see cref="Value"/>.</returns>
-        public Value GetJsonValue(int i) => _innerList[i].ConvertToClrType<Value>(GetSpannerFieldType(i));
+        public Value GetJsonValue(int i) => _innerList[i];
```

Comment:
Out of interest, why is this called GetJsonValue rather than GetProtoValue or similar?
And was this just a bug before, or is it a change in the intended meaning?

Reply:
Just a bug.  We should have called it GetProto.  I'll deprecate it in the next spanner release